<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Ajouter :</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>code_reservation :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>date_d√©but :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>heure_reservation :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_code">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="le_heure">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="le_date">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Code promotion:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codepromo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>date_fin :</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="le_date_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Modifier :</string>
    </property>
    <widget class="QLabel" name="le_newcode_res">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>52</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>code_res :</string>
     </property>
    </widget>
    <widget class="QLabel" name="le_newdate_res">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>date_res :</string>
     </property>
    </widget>
    <widget class="QLabel" name="le_newheure_res">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>heure_res :</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_modif">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="time_modif">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modify">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="code_modif">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Supprimer :</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>code_res pour supprimer :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_code_res_supp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>431</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Afficher :</string>
    </property>
    <widget class="QTableView" name="tab_reservation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>361</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>chercher le code reservation :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="le_chercher">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>300</y>
      <width>82</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Chercher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_coderechercher">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>300</y>
      <width>82</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>tri </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stat">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Statistiques</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>24</y>
      <width>1351</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Users/yassi/OneDrive/Desktop/pic1 (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>label_2</zorder>
   <zorder>le_chercher</zorder>
   <zorder>le_coderechercher</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_stat</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1323</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReservation">
    <property name="title">
     <string>Reservation</string>
    </property>
   </widget>
   <addaction name="menuReservation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
